coreneuron/CMakeLists.txt

```diff
@@ -207,8 +207,7 @@ add_dependencies(coreneuron scopmath)
 include_directories(${CORENEURON_PROJECT_SOURCE_DIR})
 add_executable(nrniv-core "apps/coreneuron.cpp")
 
-target_link_libraries(nrniv-core corenrnmech ${reportinglib_LIBRARY} ${sonatareport_LIBRARY}
-                      ${MPI_C_LIBRARIES} ${MPI_CXX_LIBRARIES})
+target_link_libraries(nrniv-core corenrnmech ${reportinglib_LIBRARY} ${sonatareport_LIBRARY})
```

Comment:
I understand about eliminating the ```{MPI_CXX_LIBRARIES}``` but I don't understand why ```${MPI_C_LIBRARIES}``` was removed

Reply:
It's ok to keep but `libcoreneuron` target already links to `MPI_C_LIBRARIES` and hence it was not needed.